tasks/mina/install.rb

```diff
@@ -9,6 +9,13 @@
   end
 
   outfile = './config/deploy.rb'
+
+  if File.exists?(outfile)
```

Comment:
can you please check all minafiles?  
https://github.com/mina-deploy/mina/blob/master/lib/mina/application.rb#L7

Reply:
But in the end, we create just this `outfile`, so I think that we souldn't check for all minafiles